### ISSUE TITLE
Eevee Shader transparent, Rigify doesnt recognize character

### ISSUE THREAD

**Issue by Gardan94** [soupday/cc3_blender_tools#139]
Hi! I just installed this Addon, the latest version. I exported an only slightly altered example-project from CC4 (Camile one) as FBX with InstaLOD active for Texture merge.
When I "Import Character" I get a brief error message (attached), than it imports the character. In Eevee however the merged material is transparent even though the material node looks just fine (in Cycles material looks better). Other than this, the whole panel doesnt appear to recognize a Character. Rigging & Animation tab, that I needed, shows "No Current Character!".

Please send help :)

![2023-03-07 21_57_49-Window](https://user-images.githubusercontent.com/127252658/223551045-c7aaaa7f-518d-420a-8ef8-1e1791605fa7.png)
![grafik](https://user-images.githubusercontent.com/127252658/223551141-0e6d0482-57d5-46ec-a72f-be950676707a.png)

**Comment by soupday** [comment]
Your color space config doesn't have Non-Color.

Are you using Agx? (https://github.com/sobotka/AgX/blob/main/config.ocio)
(Though there is an alias for Non-Color in the Agx config so it should have worked)

I'll update it to try some alternatives: i.e. "Linear", "Linear BT.709" or "Raw" instead of "Non-Color" if it fails.

**Comment by Gardan94** [comment]
Ok, it blows my mind a bit, to realize that you are the same guy, who has done the Agx thing :D Completely unrelated in my head xD I'm a concept artist, so I just blindly followed the [recommendation](https://www.elsksa.me/scientia/cgi-offline-rendering/octane-render-color-management-b3d) for Octane render, since it looked better. This Blender here is a regular one though.

I tried different options in Color Managment menu. Nothing seems to improve the material in Eevee.
Is Agx the reason the plugin won't recognize the Character to rig though? That seems to be oddly unrelated :D 

Thanks in advance for your time!

![grafik](https://user-images.githubusercontent.com/127252658/223580101-8a5c5969-e892-4974-9bc5-13dee1ece9ca.png)

**Comment by soupday** [comment]
No, that's not me. I just linked to it to see if you were using it.

Different colour spaces break Blender add-ons because they remove the expected colour spaces for sRGB or Non-Color textures. So when the add-ons try to set, say a roughness texture to Non-Color data it fails because there is no colour space for it. So I have to catch these errors and then try something else and hope it works.

I can easily detect if OCIO-ACES is in use because it sets the display device to "ACES", but AgX doesn't.

I've updated the release to try some AgX fallbacks if it fails to set the colour spaces. Namely Non-Color -> Linear BT.709.

See if that works.

**Comment by soupday** [comment]
But also if you're using the octane renderer it might not work with the materials as set up by the add-on.

**Comment by Gardan94** [comment]
Alright, I redownloaded the same version and reinstalled - sadly same thing :(
I reinstalled the Blender native ocio file and yes, everything works fine with it, even the rig functions perfectly.
